AUTHORS

```diff
@@ -983,7 +989,99 @@ Yuta Hiroto <git@about-hiroppy.com>
 Josh Leder <josh@ha.cr>
-surya panikkal <surya.com@gmail.com>
-vsemozhetbyt <vsemozhetbyt@gmail.com>
+Surya Panikkal <surya.com@gmail.com>
+Vse Mozhet Byt <vsemozhetbyt@gmail.com>
+Eugene Obrezkov <ghaiklor@gmail.com>
+Alex Lamar <alawi@vt.edu>
+Ian Kronquist <iankronquist@gmail.com>
+DavidCai <davidcai1993@yahoo.com>
+Patrick Mueller <pmuellr@nodesource.com>
+Ben Page <ben.page@openreign.com>
+Juan Soto <juan@juansoto.me>
+Allen Hernandez <allensh12@users.noreply.github.com>
+phette23 <phette23@gmail.com>
+William Luo <toyota790@gmail.com>
+justshiv <siobhan@justshiv.com>
+Kevin Donahue <kevindonahuedevelopment@gmail.com>
+Jenna Vuong <hello@jennavuong.com>
+Anchika Agarwal <anchika.agarwal@gmail.com>
+Jean Regisser <jean.regisser@gmail.com>
+Fangshi He <hefangshi@gmail.com>
+Blake Embrey <hello@blakeembrey.com>
+Alex J Burke <alex@alexjeffburke.com>
+Faiz Halde <faiz_84123@yahoo.com>
+Rumkin <dev@rumk.in>
+Arve Seljebu <arve.seljebu@gmail.com>
+Devon Rifkin <drifkin@drifkin.net>
+Doug Wade <doug@dougwade.io>
+Mohsen <mohsenekhtiari@yahoo.com>
+Marian <sorca_marian@yahoo.com>
+Justin Sprigg <IPWNDYOU@gmail.com>
+Bryan Hughes <brhugh@microsoft.com>
+Ehsan Akhgari <ehsan@mozilla.com>
+Ingvar Stepanyan <me@rreverser.com>
+Adrian Estrada <edsadr@gmail.com>
+Matt Lavin <matt.lavin@gmail.com>
+Joao Andrade <jjadonotenter@gmail.com>
+Bartosz Sosnowski <bartosz@janeasystems.com>
+Nicolas Romer <nicolasromer@gmail.com>
+David A. Wheeler <dwheeler@dwheeler.com>
+Italo A. Casas <me@italoacasas.com>
+Jonathan Montane <jon@stratus.one>
+Ilkka Myller <ilkka.myller@nodefield.com>
+kimown <kimown@163.com>
+Natanael Copa <ncopa@alpinelinux.org>
+David Halls <dahalls@gmail.com>
+Greyson Parrelli <greyson.parrelli@gmail.com>
+Daniel Bevenius <daniel.bevenius@gmail.com>
+Bryce Simonds <bryce.simonds@q2ebanking.com>
+Tushar Mathur <tusharmath@gmail.com>
+Kyle E. Mitchell <kyle@kemitchell.com>
+akki <akki@users.noreply.github.com>
+Josh Gavant <josh.gavant@outlook.com>
+sartrey <sartrey@163.com>
+Guy Fraser <guy.fraser1@gmail.com>
+Ruslan Iusupov <rus0000@users.noreply.github.com>
+Michael Wain <michael.wain@skybettingandgaming.com>
+Zach Bjornson <bjornson@stanford.edu>
+Andras <andras@kinvey.com>
+Chuck Langford <field.email@gmail.com>
+Ryan Lewis <ryanharrisonlewis@gmail.com>
+Tarun Garg <tarungarg546@gmail.com>
+Diosney Sarmiento <diosney.s@gmail.com>
+Quentin Headen <qheaden@phaseshiftsoftware.com>
+Alex Hultman <alexhultman@localhost.localdomain>
+saadq <saad@saadquadri.com>
+Hargobind S. Khalsa <khalsah@gmail.com>
+Joran Siu <joransiu@ca.ibm.com>
+Vitaly Tomilov <vitaly-t@users.noreply.github.com>
+ratikesh9 <ratikesh92@gmail.com>
+Alex Perkins <aperkin@rei.com>
+Bethany N Griggs <Bethany.Griggs@uk.ibm.com>
+Joe Esposito <joe@joeyespo.com>
+Erin Spiceland <yes@erin.codes>
+Ravindra barthwal <ravindrabarthwal@users.noreply.github.com>
+Joey Cozza <joeycozza@gmail.com>
+Vladimir de Turckheim <vlad2t@hotmail.com>
+hugnosis <hugnosis@gmail.com>
```

Comment:
Hello, Can you please make it as Taehee Kang <hugnosis@gmail.com>

thanks for asking!

2016-09-01 2:28 GMT+09:00 James M Snell <notifications@github.com>:

> In AUTHORS <https://github.com/nodejs/node/pull/8346#discussion_r77032871>
> :
>
> > +Diosney Sarmiento <diosney.s@gmail.com>
> > +Quentin Headen <qheaden@phaseshiftsoftware.com>
> > +Alex Hultman <alexhultman@localhost.localdomain>
> > +saadq <saad@saadquadri.com>
> > +Hargobind S. Khalsa <khalsah@gmail.com>
> > +Joran Siu <joransiu@ca.ibm.com>
> > +Vitaly Tomilov <vitaly-t@users.noreply.github.com>
> > +ratikesh9 <ratikesh92@gmail.com>
> > +Alex Perkins <aperkin@rei.com>
> > +Bethany N Griggs <Bethany.Griggs@uk.ibm.com>
> > +Joe Esposito <joe@joeyespo.com>
> > +Erin Spiceland <yes@erin.codes>
> > +Ravindra barthwal <ravindrabarthwal@users.noreply.github.com>
> > +Joey Cozza <joeycozza@gmail.com>
> > +Vladimir de Turckheim <vlad2t@hotmail.com>
> > +hugnosis <hugnosis@gmail.com>
>
> @hugnosis <https://github.com/hugnosis> ... how would you like your name
> listed on here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/8346/files/a00f38428b7d73e0d4a4a37a3130537f3046ab31#r77032871>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABKljXc76h3PvaZrTC3_mzQjY0wWIpKWks5qlbmzgaJpZM4JxDRm>
> .
>



-- 

Taehee Kang
E-mail <hugnosis@gmail.com>